Project.toml

```diff
@@ -15,9 +15,9 @@ SolverTools = "b5612192-2639-5dc1-abfe-fbedd65fab29"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 
 [compat]
-HSL = "0.3, 0.4"
-Krylov = "0.8, 0.9"
-LDLFactorizations = "0.8.1, 0.9, 0.10"
+HSL = "0.4"
+Krylov = "0.9"
+LDLFactorizations = "0.10"
```

Comment:
You don't change the code, so I don't see any reason to drop these

Reply:
Because nothing is tested with older version of the packages.

Reply:
HSL 0.3 doesn't support the JLL for example.
You should not allow it!

Reply:
Still, I don't think we should drop this if no coding reason.